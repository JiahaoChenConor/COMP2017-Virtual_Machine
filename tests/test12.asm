FUNC LABEL 0
    MOV STK A VAL 3
    REF REG 7 STK A
    RET
