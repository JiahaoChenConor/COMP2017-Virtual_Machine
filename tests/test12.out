invalid to put address into program counter
